Lib/test/test_tkinter/widget_tests.py

```diff
@@ -173,7 +180,7 @@ def checkImageParam(self, widget, name):
         image = tkinter.PhotoImage(master=self.root, name='image1')
         self.checkParam(widget, name, image, conv=str)
         self.checkInvalidParam(widget, name, 'spam',
-                errmsg='image "spam" doesn\'t exist')
+                errmsg=messages['no_image'].format('spam'))
```

Comment:
Similar to above, `checkInvalidParam` uses `assertRaisesRegex` internally:

```suggestion
                errmsg=r"""image "spam" does( no|n')t exist""")
```

Reply:
I removed the message catalog and replaced it with regexes, where appropriate, or cases based on tk_version.  There
weren't that many places where it was used (yet! - fingers crossed that messing with messages is over).